README.md

```diff
@@ -68,7 +68,7 @@ Make sure your Paloma Cosmos-SDK keys are stored and available on your environme
 
 Set the VALIDATOR env variable
```

Comment:
@Vizualni It is a little weird to me that we are setting `VALIDATOR` here, but _using_ `VALIDATOR` on the line above.